lib/ecommerce/client.rb

```diff
@@ -0,0 +1,45 @@
+require "ecommerce/request"
+require "ecommerce/response"
+
+module Ecommerce
+  class Client
+    attr_reader :token, :secret
+
+    def initialize(token, secret)
+      @token = token
+      @secret = secret
+    end
+
+    def authenticated?(plan)
+      get("/api/orders/#{plan}/") { |response| response.code == 200 }
+    rescue RequestError => e
+      raise e unless [401, 403].include?(e.code)
+      false
+    end
+
+    %w[get post delete put patch].each do |m|
+      define_method(m) do |path, options = {}, &block|
+        send_request(m.to_sym, path, options, &block)
+      end
+    end
+
+    private
+
+    def send_request(method, path, options, &block)
+      request  = Request.new(options.merge!({
+        method: method,
+        authorization_hash: authorization_hash,
+        url: "#{Ecommerce.configuration.url}#{path}",
+        user_agent: Ecommerce.configuration.user_agent
+      }))
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.